src/gentropy/datasource/gwas_catalog/study_index.py

```diff
@@ -647,6 +647,22 @@ def apply_inclusion_list(
             _schema=StudyIndexGWASCatalog.get_schema(),
         )
 
+    def add_no_sumstats_flag(self: StudyIndexGWASCatalog) -> StudyIndexGWASCatalog:
+        """Add a flag to the study index if no summary statistics are available.
+
+        Returns:
+            StudyIndexGWASCatalog: Updated study index.
+        """
+        self.df = self.df.withColumn(
+            "qualityControls",
+            StudyIndex.update_quality_flag(
+                f.col("qualityControls"),
+                ~f.col("hasSumstats"),
+                StudyQualityCheck.SUMSTATS_NOT_AVAILABLE,
```

Comment:
This is a bit misleading I think, as this would not mean that sumstats are not available, they are, but we did not| could not harmonise them, besides that I can see clear issue with the `qualityControls` and harmonisation qc. This get's confusing to reason what we refer in the code.